src/darker/__main__.py

```diff
@@ -132,17 +131,16 @@ def _isort_and_blacken_single_file(  # pylint: disable=too-many-arguments
         relative_path_in_rev2, revrange.rev2, root
     )
     # 1. run isort
-    if enable_isort:
-        rev2_isorted = apply_isort(
-            rev2_content,
-            relative_path_in_rev2,
-            edited_linenums_differ,
-            black_config.get("config"),
-            black_config.get("line_length"),
-        )
-    else:
-        rev2_isorted = rev2_content
-    if relative_path_in_rev2 not in black_exclude:
+    rev2_isorted = apply_isort(
+        rev2_content,
+        relative_path_in_rev2,
+        exclude.isort,
+        edited_linenums_differ,
+        black_config.get("config"),
+        black_config.get("line_length"),
```

Comment:
Maybe pass the complete `black_config` object and the function can use the values he needs?

Reply:
I've considered that, but the rest of `black_config` is Black specific configuration, and we're calling the `apply_isort()` function here. I liked to be explicit here.

Maybe an `IsortOptions` dataclass could encapsulate both of these instead.

Anyway, I decided to not touch this right now.